### ISSUE TITLE
Annotation popovers should not overlap

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1874]
![Screen Shot 2021-09-08 at 5.35.00 PM.png](https://images.zenhubusercontent.com/5d152e12c27d0c306f021863/5e04f20b-1854-418e-81d8-a58604e075e7)

I suggest we move the popovers so that they are not overlapping so they're both readable at the same time.

**Comment by mbjones** [comment]
@laurenwalker I agree the annotation popovers shouldn't overlap. One solution might be to ensure that only one popover is open at a time by automatically closing any that are open when someone opens one of them. I think this is good behaviour even if they don't overlap, as I've always found it strange to have to click again to close the popovers. Better if they close automatically if someone clicks on another one. How does that sound to you all? cc @amoeba @mpsaloha

**Comment by amoeba** [comment]
I've also found the popover behavior pretty strange. Popover claims to have a "focus" trigger mechanism that should do this for us but I haven't been able to make it work on Bootstrap 2.3.1 (our version). It does work as advertised on the 5.x series. @laurenwalker have you seen "focus" mode work?

Making sure the two popovers can both be displayed at the same time without overlapping could be tricky but having both open at the same time might be useful. I think either doing what you suggest (only show one at a time) or merging these back into a single but much wider popover might be best:

![image](https://user-images.githubusercontent.com/563/132616149-d3d02114-74cc-42d5-a213-5615804ba62c.png)

**Comment by laurenwalker** [comment]
I have seen `focus` mode work, but I think its effectiveness depends on the element type. Since these are `div`s, it can be tricky to trigger a `focus` event. I think hover/mouseover might work  better.

But I like your sketch of the single popover with info about the property and value.

**Comment by robyngit** [comment]
We can try using the FormaticUI popover module instead
